src/subside/plugins/koth/hooks/AbstractRangeHook.java

```diff
@@ -173,7 +174,7 @@ public final void tick(){
      * Reset all players. Triggered when the KoTH ends.
      */
     public final void resetAll(){
-        for(OfflinePlayer player : inRange){
+        for(OfflinePlayer player : Sets.newHashSet(inRange)){
```

Comment:
I knew I had to copy the set in some way, I didn't know newHashSet was a thing xd
Thanks! <3